pkg/providers/amifamily/bottlerocket.go

```diff
@@ -38,17 +38,42 @@ type Bottlerocket struct {
 	*Options
 }
 
-// SSMAlias returns the AMI Alias to query SSM
-func (b Bottlerocket) SSMAlias(version string, instanceType *cloudprovider.InstanceType) string {
-	arch := "x86_64"
-	amiSuffix := ""
-	if !resources.IsZero(instanceType.Capacity[v1alpha1.ResourceNVIDIAGPU]) {
-		amiSuffix = "-nvidia"
-	}
-	if instanceType.Requirements.Get(v1.LabelArchStable).Has(v1alpha5.ArchitectureArm64) {
-		arch = v1alpha5.ArchitectureArm64
+// DefaultAMIs returns the AMI name, and Requirements, with an SSM query
+func (b Bottlerocket) DefaultAMIs(version string) []DefaultAMIOutput {
+	return []DefaultAMIOutput{
+		{
+			Name:  fmt.Sprintf("bottlerocket-aws-k8s-%s%s", version, "-x86_64"),
+			Query: fmt.Sprintf("/aws/service/bottlerocket/aws-k8s-%s/%s/latest/image_id", version, "x86_64"),
```

Comment:
```suggestion
			Query: fmt.Sprintf("/aws/service/bottlerocket/aws-k8s-%s/x86_64/latest/image_id", version),
```
Same comment here and throughout the code